lib/ioki/model/passenger/ride.rb

```diff
@@ -47,6 +47,7 @@ class Ride < Base
         attribute :ticket, on: :read, type: :object, class_name: 'Ticket'
         attribute :offered_solutions, on: :read, type: :array, class_name: 'OfferedSolution'
         attribute :options, on: :create, type: :array, class_name: 'RideOption'
+        attribute :passenger_note_to_driver, type: :string, on: [:read, :create, :update]
```

Comment:
Sidenote: schalter is never updating the note for the ride, so `update` in the `on` part is a bit of _you might need it_.